integration_test/test/hls_endpoint_test.exs

```diff
@@ -119,8 +117,6 @@ defmodule Membrane.RTC.HLSEndpointTest do
                        [^rtc_engine, "hls-endpoint", ^track_id, _opts]}},
                      @tracks_added_delay
 
-      FileEndpoint.start_sending(rtc_engine, file_endpoint_id)
-
```

Comment:
Because the HLS endpoint is manually subscribed to the `file track` we should manually start the file endpoint to avoid race conditions.